DataLink.tex

```diff
@@ -370,8 +370,8 @@ \subsection{Registering \blinks\ endpoints}
 \begin{verbatim}
    ivo://ivoa.net/std/DataLink#links-1.0
 \end{verbatim}
-Note this is applicable to endpoints following any version 1.*
-of the DataLink standard, to avoid backward compatibility problems.
+Note that this (minor-versioned) URI is applicable to endpoints following any version 1.*
+of the DataLink standard.
```

Comment:
Since I think everybody agrees that the version ID should be #links-1.1, I guess this part will be revised anyway.